modules/default.nix

```diff
@@ -0,0 +1 @@
+builtins.mapAttrs (_: moduleFile: import moduleFile) (import ./list.nix)
```

Comment:
Although this is fine, we should adopt a conditional import in the future so we can import or not import files based on some condition (for instance no xorg on Darwin.)

Reply:
That would conflict with the way flakes work. If a user only wants to use some modules, they can choose to do so. Besides, we can always choose what we activate with the `mode` argument.